packages/client/src/WalletAPIClient.ts

```diff
@@ -48,6 +49,8 @@ export class WalletAPIClient extends RpcNode<
   typeof requestHandlers,
   WalletHandlers
 > {
+  public storage: Storage;
```

Comment:
Would be better to rename these to formally express these as modules for better clarity.

cf. https://github.com/LedgerHQ/wallet-api/pull/91/files#r1069163318